tutorials/independence.py

```diff
@@ -77,7 +77,22 @@
 #    :Cons: - Very slow (computationally complex)
 #           - Test statistic not very interpretable, not between (-1, 1)
 #
-# This test runs like :ref:`any other test<general indep>`.
+# An alternate fast HHG test is also available, based on a paper that proposed multivariate 
+# tests of association through univariate tests. 
+# This test obtains distance of all samples from the center of mass of the sample group and 
+# then computes the Hoeffding's D statistic (a univariate independence test).
+#
+# .. note::
```

Comment:
don't need this, it's still HHG

Reply:
Gotcha, removed the Fast HHG description. Just added under HHG pro as a 'has fast implementation' point.